Example/Podfile.lock

```diff
@@ -9,8 +9,8 @@ EXTERNAL SOURCES:
     :path: ../
 
 SPEC CHECKSUMS:
-  KeyboardSupport: f337d968304101e9247e774e333c0add2c090cb7
+  KeyboardSupport: 2c059be02d874c793b8f47bbb197804d310a4644
 
 PODFILE CHECKSUM: 84a47293f5f735c13f9b086661ea65591818a35c
 
-COCOAPODS: 1.4.0.beta.2
+COCOAPODS: 1.3.1
```

Comment:
Now that CP 1.4 is out, we'll want to make sure we're using it for the sample app.

Reply:
got it